### ISSUE TITLE
Typo on "allowusermedia" section

### ISSUE THREAD

**Issue by compulim** [w3c/webappsec-permissions-policy#153]
In [`allowusermedia`](https://wicg.github.io/feature-policy/#iframe-allowusermedia-attribute), there is a copy-and-paste typo:

> If the iframe element has an "`allow`" attribute whose value contains the token "`payment`", then the "`allowusermedia`" attribute must have no effect.

Instead of `payment`, it should be "`camera` or `microphone`".

**Comment by clelland** [comment]
Thanks -- it actually needs to be a bit more verbose than the other sections, since the intention is that "`allowusermedia`" is equivalent to `allow="camera *; microphone *`", except that specifying either `camera` or `microphone` will override the policy for just that feature, leaving the other untouched.

I'll see if I can reword it to capture that.

**Comment by foolip** [comment]
@clelland looks like this wasn't automatically closed by https://github.com/w3c/webappsec-feature-policy/pull/164.
